Refresh.GameServer/Types/GameLocation.cs

```diff
@@ -5,12 +5,18 @@ namespace Refresh.GameServer.Types;
 [XmlType("location")]
 public class GameLocation
 {
-    public static GameLocation Zero => new()
+    public static GameLocation ZeroLocation => new()
     {
         X = 0,
         Y = 0,
     };
```

Comment:
Not a fan of this rename. BCL has Vector3.Zero for example, should keep for consistency

Reply:
The Base Class Library, e.g. the base APIs that you use when you use .NET.